<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connect</class>
 <widget class="QDialog" name="Connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="addressBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="NumberButton" name="Button0">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dotButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumberButton</class>
   <extends>QPushButton</extends>
   <header location="global">numberbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
